arch/arm/src/tiva/lm/lm3s_ethernet.c

```diff
@@ -675,10 +671,11 @@ static int tiva_txpoll(struct net_driver_s *dev)
 
 static void tiva_receive(struct tiva_driver_s *priv)
 {
+  struct net_driver_s *dev = &priv->ld_dev;
```

Comment:
Ditto: Did you intend to replace all `priv->ld_dev.` below with `dev->`?

Reply:
Done